helm/hpcc/values.yaml

```diff
@@ -289,6 +289,9 @@ secrets:
   ecl: {}
     ## Category for secrets published to all components that run ecl
 
+  ecl-user: {}
```

Comment:
Possibly document how "ecl" category differs.

Also need to update the help charts to ensure the new category is generated.

Reply:
Helm chart was done long ago, I do need to test though, and will document the differences.